AHBC_MIDTERM_2019_JULY_TEAMROCKET/CategorySelectionApp.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace AHBC_MIDTERM_2019_JULY_TEAMROCKET
+{
+    class CategorySelectionApp
+    {
+
+        StoreInventory loaclInventory = new StoreInventory();
+        private int userMenuSelection { get; set; }
+        public int itemSelection { get; set; }
+        public int qtySelection { get; set; }
+
+
+        string testSelection;
+        string testQTY;
+
+        bool isValid;
+
+        List<StoreItem> currentInventory = new List<StoreItem>();
+        List<StoreItem> tempList = new List<StoreItem>();
+
+
+        public enum localCategory
+        {
+            Clothes = 1,
+            Acccessories = 2,
+            Shoes = 3,
+            Outerwaer = 4
+        }
+
+
+
+        public CategorySelectionApp(int userSelection)
+        {
+            userMenuSelection = userSelection;
+        }
+
+        public void categorySelector()
+
+        {
+            localCategory localCategory;
+
+            currentInventory = loaclInventory.GenerateStoreInventory();
+
+            Console.WriteLine("Which item would you like to purchase? (Select from the folowing numbers)");
+            if (userMenuSelection == 1)
```

Comment:
In this section, were we slowly going to add in the other enum categories?

Reply:
We were kind of debating that while working on it, Just wanna get this area build and working then worry about maybe switching it to enums